src/vma/dev/net_device_table_mgr.cpp

```diff
@@ -298,7 +307,7 @@ int net_device_table_mgr::map_net_devices()
 
 // Verify IPoIB is in 'datagram mode' for proper VMA with flow steering operation
 // Also verify umcast is disabled for IB flow
-void net_device_table_mgr::verify_ipoib_mode(struct ifaddrs* ifa)
+bool net_device_table_mgr::verify_ipoib_mode(struct ifaddrs* ifa)
```

Comment:
Before this PR this function returned void (i.e. just printing a warning) and IB device was created by the calling function. 
In this PR this function returns bool and IB device may not be created by the calling function and will not be offloaded (if returning false).
1. Please explain this change in implementation (was previous implementation a bug?)
2. If change is justified please add to message an indication that device will not be offloaded.